django/http/multipartparser.py

```diff
@@ -102,6 +102,11 @@ def __init__(self, META, input_data, upload_handlers, encoding=None):
         self._content_length = content_length
         self._upload_handlers = upload_handlers
 
+        # Limit the maximum request data size that will be parsed and stored in
+        # memory. File upload data is not included against this limit.
+        self._max_data_size = settings.DATA_UPLOAD_MAX_MEMORY_SIZE
```

Comment:
I'm don't think you need to use a instance attribute here. Referencing `settings.DATA_UPLOAD_MAX_MEMORY_SIZE` is just fine whenever you need it.